func/uk/gov/hmrc/vatapi/resources/FinancialDataResourceSpec.scala

```diff
@@ -69,6 +69,46 @@ class FinancialDataResourceSpec extends BaseFunctionalSpec {
           .thenAssertThat()
           .statusIs(404)
       }
+
+      "return code 500 when idType parameter is invalid" in {
+        given()
+          .userIsFullyAuthorisedForTheResource
+          .des().FinancialData.invalidPaymentsParamsFor(vrn, DesJsons.Errors.InvalidIdType)
+          .when()
+          .get(s"/$vrn/liabilities?from=2017-01-01&to=2017-12-31")
+          .thenAssertThat()
+          .statusIs(500)
+      }
+
+      "return code 500 when idNumber parameter is invalid" in {
+        given()
+          .userIsFullyAuthorisedForTheResource
+          .des().FinancialData.invalidPaymentsParamsFor(vrn, DesJsons.Errors.InvalidIdNumber)
+          .when()
+          .get(s"/$vrn/liabilities?from=2017-01-01&to=2017-12-31")
+          .thenAssertThat()
+          .statusIs(500)
+      }
+
+      "return code 400 when regime type parameter is invalid" in {
```

Comment:
Test description states 400 but we're asserting for 500, minor update to the test description needed.